dist/access_check/MODULARITY.md

```diff
@@ -0,0 +1,44 @@
+# Rules for Modularity
+
+## Definition of a Module
+
+* Whatever resides in a subdirectory under `src/` is considered to be a module.
+
+* The module names are pre-configured as a list of subdirectories under `src/`, with some exclusions like `include`, `checksum`, `os*`, etc. These exclusions are because these directories don’t really contain modular code. If the module name, as deduced by any method, is not in this list, there is no associated module, and the content is exempt from modularity checks.
+
+* Modules can have aliases. `block_cache` refers to `blkcache`, or `connection` is the same as `conn`.
```

Comment:
```suggestion
* Modules can have aliases including but not limited to`block_cache` which refers to `blkcache`, or `connection` is the same as `conn`.
```

Reply:
Yes that's why the "not limited to" is important. It's just to give a heads up with examples.